client/lib/graph/resolvers/post.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Internal dependencies
+ */
+import { getSitePost } from 'state/posts/selectors';
+import { requestSitePost } from 'state/posts/actions';
+import { refreshByUid } from './utils';
+import createPostStatResolver from './post-stat';
+
+export const resolvePost = ( store, post ) => {
+	if ( ! post ) {
+		return post;
+	}
+	return {
+		...post,
+		stat: ( { stat } ) => createPostStatResolver( store )( {
+			siteId: post.site_ID,
+			postId: post.ID,
+			stat
+		} )
+	};
+};
+
+const createResolver = store => ( args, { uid } ) => {
```

Comment:
I think it is perhaps at this line, but I'm receiving an error when trying to use GraphiQL with the following query:

```graphql
query {
  posts( siteId: 2916284 ) {
    title
  }
}
```

```json
{
  "data": {
    "posts": null
  },
  "errors": [
    {
      "message": "Cannot read property 'uid' of undefined",
      "locations": [
        {
          "line": 16,
          "column": 3
        }
      ],
      "path": [
        "posts"
      ]
    }
  ]
}
```